pages/app/models/refinery/page.rb

```diff
@@ -291,8 +291,13 @@ def to_refinery_menu_item
     #    ::Refinery::Page.first.content_for(:body)
     #
     # Will return the body page part of the first page.
-    def content_for(part_title)
-      part_with_title(part_title).try(:body)
+    def content_for(part_slug)
+      begin
+        part_with_slug(part_slug).try(:body)
+      rescue
```

Comment:
..rescue _what_ exactly?  Should this only rescue `ActiveRecord::RecordNotFound`?